### PR TITLE
Elementor compatibility changes.

### PR DESCRIPTION
**Pull request by swapnilde**
### Description
Changes in this PR.
- Added compatibility to Elementor core 3.5.0.
- Fixed PHP warning of undefined array key on responsive controls in all widgets.
- Changed minimum required elementor version to 3.1.0
- Run update related commands.

### Types of changes

Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,6 +137,10 @@ This same applies when you are creating your Header/Footer using this plugin.
 5. Default widgets available with Elementor Header & Footer Builder.
 
 ## Changelog ##
+### 1.6.6 ###
+- Improvement: Added compatibility to Elementor core version 3.5.0. Now to work with UAE, Elementor minimum version 3.1.0 or higher will be required.
```
Comment (sushmak02) [inline README.md:141]: UAE?

File: readme.txt [hunk 2]
```diff
@@ -5,7 +5,7 @@ Donate link: https://www.paypal.me/BrainstormForce
 Requires at least: 4.4
 Requires PHP: 5.4
 Tested up to: 5.8
```
Comment (sushmak02) [inline readme.txt:7]: 5.8.2

File: README.md (hunk 1 above)
Comment (swapnilde) [reply README.md]: Done.

File: readme.txt (hunk 2 above)
Comment (swapnilde) [reply readme.txt]: Done.

File: inc/widgets-manager/widgets/class-navigation-menu.php [hunk 3]
```diff
@@ -391,20 +391,20 @@ protected function register_general_content_controls() {
 						'{{WRAPPER}} .hfe-nav-menu__toggle,
 						{{WRAPPER}} .hfe-nav-menu-icon' => '{{VALUE}}',
 					],
-					'default'              => 'center',
```
Comment (sushmak02) [inline inc/widgets-manager/widgets/class-navigation-menu.php:394]: confirm, why this default statement is removed?

Comment (swapnilde) [reply inc/widgets-manager/widgets/class-navigation-menu.php:394]: This `'default' => 'center',` was written twice so I removed the duplicate one.
This statement is already written on 370 line.

**Review by sushmak02 — APPROVED** [review]

**Pull request merged**